### DISCUSSION TITLE
doris支持bitmap_hash64？

### DISCUSSION THREAD

**Discussion by xiaoL1211** [apache/doris#12352] [A - General / Q&A] [answered]
现在bitmap_hash计算32位的会有冲突
请问bitmap_hash64 这个支持了吗？
或者murmur_hash3_64？

**Comment by LemonLiTree** [comment, marked as answer]
bitmap_hash64已经在排期开发，另外目前int64做bitmap计算效率比较低，这块的优化也在排期了，都会在后续版本发布。

**Reply by LemonLiTree** [reply]
可以关注下：#4718

**Reply by xiaoL1211** [reply]
您好。关于bitmap_hash64预计什么时候可以发布

**Reply by LemonLiTree** [reply]
Maybe 1.2

**Reply by xiaoL1211** [reply]
好的，我先手动改下源码编辑试试murmur_hash2_64

**Reply by zy-kkk** [reply]
你好，bitmap_hash64已完成开发，预计会在1.2版本发布，可以查看  #12992
